server.go

```diff
@@ -41,18 +40,22 @@ type Server struct {
 }
 
 // NewServer returns a fresh Server tied to a given Router.
-func NewServer(r *network.Router, pkey kyber.Scalar, s network.Suite) *Server {
+// If where is "", the server will write its database to the default
+// location. If where is != "", it is considered a temp dir, and the
+// DB is deleted on close.
+func newServer(where string, r *network.Router, pkey kyber.Scalar, s network.Suite) *Server {
```

Comment:
`s/where/delDb`

Reply:
I missed that one before, pushign a new change to fix it.